tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests import (  # noqa
```

Comment:
Not sure why it's not showing, but this was what I was suggesting might be unnecessary - it wasn't required on the travis config branch anyway.